core/graphelier-service/models/orderinfo_test.go

```diff
@@ -0,0 +1,69 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var newOrderMessage *Message = &Message{Direction: -1, Instrument: "test", Type: NewOrder, OrderID: 1, Price: 100, ShareQuantity: 10, Timestamp: 1, SodOffset: 1}
+var modifyMessage *Message = &Message{Direction: -1, Instrument: "test", Type: Modify, OrderID: 1, Price: 100, ShareQuantity: 2, Timestamp: 2, SodOffset: 2}
+var ignoreMessage *Message = &Message{Direction: -1, Instrument: "test", Type: Ignore, OrderID: 1, Price: 100, ShareQuantity: 10, Timestamp: 3, SodOffset: 3}
+var executeMessage *Message = &Message{Direction: -1, Instrument: "test", Type: Execute, OrderID: 1, Price: 100, ShareQuantity: 8, Timestamp: 4, SodOffset: 4}
```

Comment:
The `MakeMsg` factory method would help distinguish the meaningful values between each message.